README.md

```diff
@@ -312,6 +324,133 @@ EOF
+metadata:
+  labels:
+    app: notification-manager
+  name: default-webhook-secret
+  namespace: kubesphere-monitoring-system
+type: Opaque
+
+---
+apiVersion: notification.kubesphere.io/v1alpha1
+kind: WebhookConfig
+metadata:
+  name: default-webhook-config
+  namespace: kubesphere-monitoring-system
+  labels:
+    app: notification-manager
+    type: default
+spec:
+  url: http://127.0.0.1:8080/
+  httpConfig: 
+    bearerToken
+      key: password
+      name: default-webhook-secret
+    tlsConfig:
+      rootCA:
+        key: ca
+        name: default-webhook-secret
+      clientCertificate:
+        cert:
+          key: cert
+          name: default-webhook-secret
+        key:
+          key: key
+          name: default-webhook-secret
+      insecureSkipVerify: false
+
+---
+apiVersion: notification.kubesphere.io/v1alpha1
+kind: WebhookReceiver
+metadata:
+  name: global-webhook-receiver
+  namespace: kubesphere-monitoring-system
+  labels:
+    app: notification-manager
+    type: global
+EOF
+```
+
+> - The `rootCA` is the server root certificate.
+> - The `certificate` is the clientCertificate of client.
+> - The format of bearerToken is `Authorization <bearerToken>`.
+
+#### Deploy the default DingTalkConfig and a global DingTalkReceiver
+
+```
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+data:
+  appkey: ZGluZ2Jla3UxR2enAyeHQ=
+  appsecret: dnRFNWt2RWppOWdiZF9x
+  webhook: aHR0cHM6Ly9vYXBpLmRpbmd0YWxrLmNvbS9yb2JvdC9zZW5kP2FjY2Vzc190b2tlbj0zNjUxO
+  secret: U0VDZjJiMTkyOGUwOGY5ZjM4YzIwMmZGNiN2VhMjk1MTMyNDI0YTgxMDljMjFkYzYwNGU3MDkzNQ==
+kind: Secret
+metadata:
+  labels:
+    app: notification-manager
+  name: default-dingtalk-secret
+  namespace: kubesphere-monitoring-system
+type: Opaque
+
+---
+apiVersion: notification.kubesphere.io/v1alpha1
+kind: DingTalkConfig
+metadata:
+  name: default-dingtalk-config
+  namespace: kubesphere-monitoring-system
+  labels:
+    app: notification-manager
+    type: default
+spec:
+  conversation:
+    appkey: 
+      key: appkey
+      name: default-dingtalk-secret
+    appsecret:
+      key: appsecret
+      name: default-dingtalk-secret
+    chatid: chat894f9f4d634eb283933af6c7102977b2
+  chatbot:
+    webhook:
+      key: webhook
+      name: test-user-dingtalk-secret
+    keywords: 
+    - kubesphere
+    secret:
+      key: secret
+      name: test-user-dingtalk-secret
+
+---
+apiVersion: notification.kubesphere.io/v1alpha1
+kind: DingTalkReceiver
+metadata:
+  name: global-dingtalk-receiver
+  namespace: kubesphere-monitoring-system
+  labels:
+    app: notification-manager
+    type: global
+EOF
+```
+
+> - DingTalkReceiver can both send messages to `conversation` and `chatbot`.
+> - If you want to send messages to conversation, the application used to send messages to conversation must have the authority `Enterprise conversation`, and the ip which notification manager used to send messages must be in the white list of the application. Usually, it is the ip of the Kubernetes nodes.
```

Comment:
ip => IP
it is the ip of the Kubernetes nodes. =>
it is the IP of Kubernetes nodes, you can simply add all Kubernetes nodes to the white list.